### DISCUSSION TITLE
Can I use tkPDFViewer with custom tkinter?

### DISCUSSION THREAD

**Discussion by Tenimes** [TomSchimansky/CustomTkinter#1354] [Q&A] [answered]
Hello I want to make a project working with pdfs so I have to open pdf file in custom tkinter window. But I don't know that will work.

**Comment by abhaymakes** [comment, marked as answer]
Yes, you can use [tkPDFViewer](https://pypi.org/project/tkPDFViewer/) to display PDFs in customtkinter. Below is the example code:

```python
from tkPDFViewer import tkPDFViewer as pdf 
from customtkinter import * 

root = CTk() 

variable1 = pdf.ShowPdf() 

variable2 = variable1.pdf_view(root,pdf_location=r"location",width=50,height=100) variable2.pack() 

root.mainloop()
```

**Comment by Akascape** [comment]
@Tenimes Use this new CTkPDFViewer: 
https://github.com/Akascape/CTkPDFViewer

![example](https://github.com/Akascape/CTkPDFViewer/assets/89206401/1324243e-da47-4bd8-af46-cded53cb7b51)

_Don't use tkPDFViewer because it doesn't work anymore with the new versions of its dependencies. Moreover, the owner is also not active._

**Reply by Tenimes** [reply]
This is a good feature, thank you.
